#1

yukimatsumoto0 opened the pull request:
html_作成

yukimatsumoto0 committed:
11/08_html作成

yukimatsumoto0 committed:
11/8_file_add

ykyk1218 commented on index.html:
```diff
@@ -0,0 +1,481 @@
+                    </a>
+
+                    <div class="header_main_btn">
+                        <li class="header_main_btn_first">
```
ul タグを親に持たない liタグは存在しないので、ulタグをつけてあげてください

yukimatsumoto0 replied:
追記しました！

ykyk1218 commented on index.html:
```diff
@@ -0,0 +1,481 @@
+            </div>
+    </div>
+
+        <div class="header_main">
```
インデントがずれてしまっています 😔

ykyk1218 commented on index.html:
```diff
@@ -0,0 +1,481 @@
+                </div>
+        </div>
+
+            <nav class="header_main_nav">
```
インデントが... 😔

yukimatsumoto0 replied:
すいません！beautifyかけわすれてました。。

ykyk1218 commented on index.html:
```diff
@@ -0,0 +1,481 @@
+                </ul>
+            </nav>
+
+<main>
```
body を一番の先頭にしてその中に入るものは全てインデントをつけるようにしてあげてください

ykyk1218 commented on index.html:
```diff
@@ -0,0 +1,481 @@
+        </picture>
+    </div>
+
+    <h1 class="main_top_text">
```
h1 の内容がちょっと多いですね
このページにとって一番重要なキーワードに絞ってつけてあげましょう

yukimatsumoto0 replied:
こちらはh1クラス内のテキスト量が多いので絞ったほうがいい、という認識でOKでしょうか？
その形で修正しました！

ykyk1218 commented on index.html:
```diff
@@ -0,0 +1,481 @@
+
+    </h1>
+
+    <div class="contents_base">
```
この div の中でもう少しセクションでくぎれるところがあるかと思いますので section タグを使ってください

yukimatsumoto0 replied:
section追記しました！

ykyk1218 commented on index.html:
```diff
@@ -0,0 +1,481 @@
+                    </div>
+            </div>
+
+        <div class="contact-unit">
```
この中にも section が欲しいです

ykyk1218 commented on index.html:
```diff
@@ -0,0 +1,481 @@
+
+                    <div class="contents_second_inner_footer-link">
+                        <p class="contents_second_inner_footer-link_title">ネットの高校とは</p>
+                        <ul class="contents_second_inner_footer-link_list">
```
ul の閉じタグがなさそうです 😇

ykyk1218 commented on index.html:
```diff
@@ -0,0 +1,481 @@
+
+        <div class="header_main">
+                <div class="header_main_logo">
+                    <a href=""><img src="./img/logo_header.svg" alt="">
```
全体的に alt が空になってしまっているので、適切なテキストを入れてあげてください 😄

yukimatsumoto0 committed:
11/14fix

yukimatsumoto0 commented:
fix/記述ミス複数
fix/l128付近_section指定
fix/l260付近_section指定
add/imgタグ内のalt記述

ykyk1218 commented on index.html:
```diff
@@ -0,0 +1,517 @@
+
+<body>
+    <div class="contents">
+        <div class="header_sub">
```
headerタグを使ってヘッダーになりそうなところを囲んであげてください

ykyk1218 commented:
とりあえず今回(htmlのレビュー)は大丈夫ですが、cssのクラス名が順番を意識したようなクラス名になっているのが気になります。
02とかfirstとかsecondとか
この場合コンテンツの順番を入れ替えたときに、クラス名と実体が伴わなくなるので名前の付け方としてはあまり良くありません。
cssの方では名前を少し考えてプルリクを出してみてください🙂

yukimatsumoto0 committed:
12/5途中

yukimatsumoto0 committed:
12/6_途中

yukimatsumoto0 committed:
12/13 commit

ykyk1218 commented:
<img width="1648" alt="ネットの高校とは___N高等学校_S高等学校_設置認可申請中__通信制高校_広域・単位制_" src="https://user-images.githubusercontent.com/830861/102444560-d63e8800-406c-11eb-8e8c-ec094fc851a0.png">

ykyk1218 commented:
コンテンツの幅に関しては「ネット高校の特徴」も画面いっぱいに広がってしまっているので、こちらも調整お願いします

<img width="1671" alt="ネットの高校とは___N高等学校_S高等学校_設置認可申請中__通信制高校_広域・単位制_" src="https://user-images.githubusercontent.com/830861/102444869-87ddb900-406d-11eb-9e43-752407350e53.png">

yukimatsumoto0 committed:
12/26　update

yukimatsumoto0 commented:
fix/ヘッダーロゴmax-width修正
fix/トップ画上テキスト位置修正
fix/"content__header__title"位置修正
fix/"content__header__title::after"ライン位置修正
fix/content__header__desc左右位置修正

ykyk1218 commented:
<img width="1539" alt="ネットの高校とは___N高等学校_S高等学校_設置認可申請中__通信制高校_広域・単位制_" src="https://user-images.githubusercontent.com/830861/103161272-8f961e00-4822-11eb-9c39-f6bffc32d31c.png">

ykyk1218 commented:
<img width="1369" alt="ネットの高校とは___N高等学校_S高等学校_設置認可申請中__通信制高校_広域・単位制_" src="https://user-images.githubusercontent.com/830861/103161329-6de96680-4823-11eb-96fc-68c7276d3434.png">

ykyk1218 commented on index.html:
```diff
@@ -0,0 +1,628 @@
+    </div>
+
+    <div class="breadcrumb">
+        <ul class="breadcrumb__inner">
+            <li class="breadcrumb__items">
+                <a class="breadcrumb__items__link" href="#">
+                    トップ
+                </a></li>
+            <p>></p>
+            <li class="breadcrumb__items">ネットの高校とは</li>
+        </ul>
```
html の時の指摘もれなのですが、ul タグの中に p タグが入ってしまっています。
ul タグの直下は li タグ以外は禁止されているので、他の方法で対応をお願いします！

ykyk1218 commented:
全体的にボタンの hover アクションがついていないようなので、こちら追加をお願いします！

yukimatsumoto0 committed:
12/27 commit
fix/ヘッダーナビの左右位置修正
fix/コンタクト"資料請求"列ボタンの文字サイズ修正
fix/コンタクト"電話番号"列テキストの位置、上下間隔修正
fix/html__l427付近<ul>内の記述ミス修正
